command/stats_test.go

```diff
@@ -0,0 +1,89 @@
+package command_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/ghttp"
+	. "github.com/openebs/maya/command"
+	"os"
+)
+
+var _ = Describe("GetVolAnnotations", func() {
```

Comment:
That's my doubt also, because GetVolDetails is called in GetVolAnnotations itself, so this function is tested along with the GetVolAnnotations, that's what i think, correct me if i wrong. Now do i need to write test (mock) for GetVolDetails also other than this? @prateekpandey14 is it required?